app/src/main/java/org/schabi/newpipe/settings/AppearanceSettingsFragment.java

```diff
@@ -31,7 +32,7 @@ public boolean onPreferenceChange(final Preference preference, final Object newV
 
             if (!newValue.equals(startThemeKey) && getActivity() != null) {
                 // If it's not the current theme
-                getActivity().recreate();
+                ActivityCompat.recreate(requireActivity());
```

Comment:
Isn't `recreate()` also begin used in `ServicePlayerActivity` and `MainActivity`?

Reply:
This was the only  usage of `recreate` I found.